docs/site/Extension-life-cycle.md

```diff
@@ -0,0 +1,71 @@
+---
+lang: en
+title: 'Extension life cycle'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Extension-life-cycle.html
+---
+
+## Extension life cycle
+
+As described in [Life cycle](Life-cycle.md), a LoopBack
+[Application](Application.md) has its own life cycles at runtime. Corresponding
+events such as `start` and `stop` are emitted upon the state change.
```

Comment:
Are there any events apart from `start` and `stop` that can be observed? If no, should make it very clear about the fact.